styles.css

```diff
@@ -623,3 +632,453 @@ button:disabled {
+    line-height: 28px;
+  }
+
+  .card-works {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    place-content: center;
+    justify-content: center;
+    row-gap: 37px;
+    column-gap: 24px;
+    height: max-content;
+    width: 80.2vw;
+  }
+
+  .card {
+    height: 435px;
+    width: 25.62vw;
+  }
+
+  .custom-card1 {
+    border: 8px #0d151e;
+    background-image: url("images/Img Placeholder.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .non-content {
+    display: none;
+  }
+
+  .custombt {
+    position: absolute;
+    top: 364px;
+    left: 0;
+    height: 58px;
+  }
+
+  .custom-card2 {
+    background-image: url("images/placeholder1.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .custom-card3 {
+    background-image: url("images/placeholder2.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .custom-card-shadow {
+    background-image: linear-gradient(180.45deg, #26262600 0.75%, #262626e6 84.18%), url("images/Img Placeholder.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .portrait {
+    position: absolute;
+    bottom: 8px;
+    gap: 24px;
+    width: 25.62vw;
+    height: auto;
+    padding-top: 0;
+    margin-bottom: 8px;
+    left: 0;
+  }
+
+  .profesional-art {
+    width: 23.4vw;
+    height: auto;
+    margin-left: 16px;
+  }
+
+  .card-description {
+    width: 23.4vw;
+    height: max-content;
+    margin-left: 16px;
+  }
+
+  .tags {
+    width: 16.38vw;
+    margin: 0 0 0 16px;
+  }
+
+  .about-me-section {
+    background: url("images/Frame-footer-desktop.png");
+    background-position: right 5vw;
+    background-repeat: no-repeat;
+    background-size: 48%;
+    background-color: #262626;
+    width: 100vw;
+    height: max-content;
+    align-items: flex-start;
+  }
+
+  .about-me {
+    width: 39.3vw;
+    height: max-content;
+    margin-left: 142px;
+    padding: 0;
+  }
+
+  .about-title {
+    padding: 0;
+    width: 39.3vw;
+    line-height: 82px;
+    color: #f55800;
+    margin-top: 130px;
+  }
+
+  .description-about {
+    width: 39.3vw;
+    font-size: 20px;
+    line-height: 32px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .resume-button {
+    margin-left: 0;
+    margin-bottom: 139px;
+  }
+
+  .solidhr1 {
+    width: 100vw;
+    margin-left: 0;
+    margin-bottom: 0;
+  }
+
+  .skills {
+    flex-direction: row;
+    align-items: flex-start;
+    width: 80.37vw;
+    height: max-content;
+    margin: 0;
+    margin-left: 9.79vw;
+    gap: 10.55vw;
+  }
+
+  .about-me-sectiones {
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    width: 20.1vw;
+    height: max-content;
+    margin-top: 130px;
+  }
+
+  .about-languages {
+    align-items: flex-start;
+    padding: 0 12px;
+    gap: 8px;
+    width: 186px;
+    height: 216px;
+  }
```

Comment:
- [ ] Please complete the list of technical skills, take in mind there are different skills in each one, your design should be similar to the Figma design 👇🏽 
- [ ] Kindly use the styles described in the Figma design especially the family fonts to be similar to the original design as much as possible.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/213596405-be5f803a-1b0b-4b18-8263-47a87a5928f2.png) | ![image](https://user-images.githubusercontent.com/58642949/213596484-f500f214-3604-49d2-99ab-4b44b363b48f.png) |